src/routes/Shop.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import FiltersSideBar from "../components/shop-page/FiltersSideBar";
+import ProductPageCard from "../components/shop-page/ProductPageCard";
+import SortProductsDropdown from "../components/shop-page/SortProductsDropdown";
+
+function Shop() {
+  return (
+    <main
+      style={{ top: "5rem", minHeight: "180vh" }}
+      className="relative w-full justify-end flex"
+    >
+      <FiltersSideBar />
+
+      <div className="products-display-container">
+        <div className="flex w-full justify-end">
+          <SortProductsDropdown />
+        </div>
+
+        <div style={{ gap: "4rem" }} className="grid grid-cols-3">
+          {/* <ProductCard product={{}} />
```

Comment:
You can remove these comments as well (line no. 20 to 26)

Reply:
okay i will remove the commented code